pages/about/index.vue

```diff
@@ -5,6 +5,48 @@
       image="https://assets-cdn.sums.su/YU/website/img/Roses/Hero_Banner_About.webp"
     />
     <div class="container mx-auto py-28">
+      <div class="flex flex-col gap-6 pb-28">
+        <h2 class="text-5xl xcond font-bold">THE BATTLE IS COMING HOME</h2>
+        <div class="flex flex-col gap-4">
+          <p>
+            Roses is an annual sporting varsity between the University of York
+            and Lancaster University. But it wasn’t always a sports contest. It
+            began way back in medieval times as a war for the throne of England,
+            between the Houses of York and Lancaster. When that war was fought
+            and won, a new one began... Now, our battlefield is the playing
+            field, fought between the University of York and Lancaster
+            University. Each This year, we take turns to host this epic battle
+            and 2025 is our turn. Wwe’ll be hosting hundreds of fixtures between
+            Friday 2 and Sunday 4 May, from American Football and Cheerleading
+            to Debating and Octopush!
+          </p>
+          <p>
+            But it won’t just be sports fixtures; across the weekend, there will
+            be a range of events to attend across campus and in our venues. So
+            if you aren’t a massive sports fan, you can still get involved in
+            the Roses buzz! This year is the 60th anniversary of the
+            competition, so it’s a special one for everyone!
+          </p>
+        </div>
+      </div>
+      <div class="flex flex-col gap-6 pb-28">
+        <h2 class="text-5xl xcond font-bold">SHOW US HOW YOU’RE CELEBRATING</h2>
+        <div class="flex flex-col gap-4">
+          <p>
+            We want to hear from you throughout the weekend. Whether you’re
+            celebrating a win, cheering on Team York from the sidelines, or
+            enjoying a well-earned pizza at Courtyard, let us know!
+          </p>
+          <p>
+            Tag @yorkunisu in your Instagram posts and we’ll reshare some of our
```

Comment:
Can we link this to the insta and add the link styling
```suggestion
            Tag <a href="https://instagram.com/yorkunisu" target="_blank" class="text-roses-red underline">@yorkunisu</a> in your Instagram posts and we’ll reshare some of our
```